cmd/limactl/hostagent.go

```diff
@@ -122,5 +122,10 @@ func initLogrus(stderr io.Writer) {
 	logrus.SetOutput(stderr)
 	// JSON logs are parsed in pkg/hostagent/events.Watcher()
 	logrus.SetFormatter(new(logrus.JSONFormatter))
-	logrus.SetLevel(logrus.DebugLevel)
+	// HostAgent logging is one level more verbose than the start command itself
```

Comment:
Isn't this a bug?

Reply:
Idk, I thought it was intentionally set to always log at DEBUG because the output goes to `ha.stderr.log` and doesn't pollute the terminal output: e88241ec8fa2f82f3b8322759c3c627f141eb141.

Which made sense to me: I don't normally want the output of `limactl --debug start` in the terminal, but I only look at `ha.stderr.log` when something is wrong, so always getting DEBUG output there is useful, so you don't have to restart your VM with `--debug` to get full logs.

So assuming this was intentional, I extended it to include TRACE level output when you run with `--debug`, as there is no way to specify the logging level for HA separately from the main logging level.

Reply:
IIRC that was just my silly workaround to avoid bother implementing `--log-level` propagation 🤦

Reply:
> IIRC that was just my silly workaround to avoid bother implementing --log-level propagation

Log-level propagation is implemented (and I tested that it works when you remove the explicit override the DEBUG in the HA code): https://github.com/lima-vm/lima/blob/5b9df0e0d7b0b471fb5e72d48625a378dc579ea4/pkg/start/start.go#L127-L129

I continue to think that always having debug logging active in `ha.stderr.log` is useful, even though it feels inconsistent. I just don't want to include DNS lookups in it by default.

So on a practical level this PR works for me. If we want to regularize the settings, I think we need a separate log-level for HA that should default to DEBUG. But this is just adding complexity for little benefit.

Reply:
> Log-level propagation is implemented 

I just checked, and the propagation was already implemented by the time the override was added. So it still feels intentional to me. 😄

Reply:
Propagation of custom log level is not implemented

Reply:
Sorry, you are right, I've forgotten my own coding design 😞